Artifacts.toml

```
@@ -1,3 +1,11 @@
+[QSMDB]
+git-tree-sha1 = "5c82e5ff3aa99f2aa660c9ffb3caa57ee535cc21"
+lazy = true
+
+     [[QSMDB.download]]
+     sha256 = "3f69aae0fca02d74764f709c622388c564e6e144a8b11347dadef5725a65b0a2"
+     url = "https://github.com/oscar-system/Oscar.jl/releases/download/archive-tag-1/qsmdb.tar.gz"     
```

```suggestion
     url = "https://github.com/oscar-system/Oscar.jl/releases/download/archive-tag-1/qsmdb-v1.tar.gz"     
```
To make it possible to change this tar at a later time while still keeping the old version installable, it is good practice to add some kind of version number for the artifact. See e.g. https://github.com/thofma/Hecke.jl/blob/0868996ac2585dc60b5d294a93c9ce94eac24065/Artifacts.toml#L6

---

but, is this different from the archive-tag-1 ? where here we use the 1 as the version?

---

That's possible as well, if you want to create a new tag for every new version of the db.